openpype/hosts/aftereffects/api/extension/js/main.js

```diff
@@ -167,7 +168,8 @@ function main(websocket_url){
         log.warn('Server called client route "import_file_with_dialog":', data);
         var escapedPath = EscapeStringForJSX(data.path);
         return runEvalScript("importFileWithDialog('" + escapedPath +"', " +
-                                         "'" + data.item_name + "')")
+                                         "'" + data.item_name + "', " +
+                                         "'" + data.fps + "')")
```

Comment:
fps should be added in the `import_options`, not as a new argument of the function